lib/rules/jsx-handler-names.js

```diff
@@ -24,23 +24,42 @@ module.exports = {
       type: 'object',
       properties: {
         eventHandlerPrefix: {
-          type: 'string'
+          type: ['string', 'boolean']
         },
         eventHandlerPropPrefix: {
-          type: 'string'
+          type: ['string', 'boolean']
         }
       },
       additionalProperties: false
     }]
   },
 
   create(context) {
+    function isPrefixDisabled(prefix) {
+      return (
+        typeof prefix === 'boolean' &&
+        prefix === false
+      );
+    }
+
     const configuration = context.options[0] || {};
-    const eventHandlerPrefix = configuration.eventHandlerPrefix || 'handle';
-    const eventHandlerPropPrefix = configuration.eventHandlerPropPrefix || 'on';
+    const eventHandlerPrefix = isPrefixDisabled(configuration.eventHandlerPrefix) ?
+      null :
+      configuration.eventHandlerPrefix || 'handle';
+    const eventHandlerPropPrefix = isPrefixDisabled(configuration.eventHandlerPropPrefix) ?
+      null :
+      configuration.eventHandlerPropPrefix || 'on';
+
+    if (!eventHandlerPrefix && !eventHandlerPropPrefix) {
+      throw new Error('Both \'eventHandlerPrefix\' and \'eventHandlerPropPrefix\' cannot be false');
```

Comment:
```suggestion
      throw new TypeError('Both \'eventHandlerPrefix\' and \'eventHandlerPropPrefix\' cannot be false');
```

however, this should be enforced via the rule schema, not via runtime JS.

In other words, the schema should allow:
 - string or false
 - false or string
 - string or string

but not "false or false", and no `true`s.

Reply:
Done